src/components/accordion/__styles__/index.ts

```diff
@@ -12,15 +13,14 @@ export const ElAccordionItem = styled.div`
   display: flex;
   justify-content: space-between;
   align-items: center;
-  cursor: pointer;
   padding: 0.875rem 0;
   border-bottom: 1px solid var(--color-grey-100);
 
   &:has(+ .${elIsActive}) {
     border-bottom: none;
   }
 `
-export const ElAccordionTitle = styled.div`
+export const ElAccordionTitle = styled(unstyledAnchor)`
```

Comment:
See my comment on unstyledAnchor. I'd just do styled.a and add any overides for the pseudo classes you need